### ISSUE TITLE
Why does snuffles throw an error on e.g. 404

### ISSUE THREAD

**Issue by jhilden** [railslove/snuffles#13]
My expectation would be that snuffles does not change the default fetch behavior that I'm used to:  https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch

**Comment by Plsr** [comment]
True, we're pretty opinionated on this part. As to _why_ it throws an error: This is simply code taken as it was used in one of our project, there was no real decision behind it.

I am using it this way in another project right now and it is somewhat comfortable to just be able to catch an error in my api wrapper instead of having to inspect the response. There are also a fair amount of people around the internet expecting this behaviour from fetch in the first place.

However, I do not have a strong opinion on the matter. If we were to change that, that would probably mean the release of `v2.0.0` though.

Anyways, we should definitely document this behaviour in the Readme for v1, since it might catch some people off guard.
